consolidate software overview/development info (#8243)
acl-org/acl-anthology: hugo/content/info/development.md

date: "2025-09-26"
---

### Software

> The ACL Anthology is built using completely open-source software, with further developments and customization done by the ACL Anthology staff.

Reviewer: ```suggestion
The ACL Anthology is built from open-source software.
```

> The current Anthology website is statically generated using the [Hugo](https://gohugo.io/) framework, with the layout being designed using the [Bootstrap](https://getbootstrap.com/) library.

Reviewer: ```suggestion
The Anthology website uses the [Hugo](https://gohugo.io/) framework to generate a static website that makes heavy use of the [Bootstrap](https://getbootstrap.com/) library for a modern design. We use [Font Awesome](https://fontawesome.com/) for icon fonts.
```

[Font Awesome](https://fontawesome.com/) is used as the icon font.

> The [source code](https://github.com/acl-org/acl-anthology/) for the Anthology is available on GitHub at <https://github.com/acl-org/acl-anthology/>.

Reviewer: ```suggestion
Our source code and data [are available on Github](https://github.com/acl-org/acl-anthology/).
```
